packages/web-components/src/components/ic-side-navigation/ic-side-navigation.css

```diff
@@ -11,6 +10,7 @@
     --ic-transition-duration-slow
   );
   --side-navigation-width: 20rem;
+  z-index: var(--ic-z-index-sidebar);
```

Comment:
Think we just need these updating to the new name of the variable :)

Reply:
D'oh!